### ISSUE TITLE
Shortcut to increase font size using keyboard with numpad

### ISSUE THREAD

**Issue by JAStanton** [Microsoft/vscode#9630]
- VSCode Version: Code 1.3.1 (e6b4afa53e9c0f54edef1673de9001e9f0f547ae, 2016-07-12T13:31:10.994Z)
- OS Version: Darwin x64 15.5.0

While using the `+` and `-` keys on the numpad, `CMD -` will decrease font but `CMD +` will not increase the font size. `CMD +` Does work for the `+` button near the delete key however.

**Comment by alexandrudima** [comment]
@JAStanton What keyboard layout do you have configured?

**Comment by JAStanton** [comment]
Standard apple wired keyboard with numpad:

![mb110ll-1](https://cloud.githubusercontent.com/assets/645300/17793275/d9899546-655a-11e6-9b8b-efe4684d7b41.jpg)

**Comment by alexandrudima** [comment]
I have validated that we now (i.e. after PR #22894 aka #17521 -- in today's Insiders build) appear to behave correctly for numpad keys. I have attached a standard QWERTY keyboard to a mac mini and configured OSX to use the US layout. Pressing numpad keys appear to be interpreted correctly:

![image](https://cloud.githubusercontent.com/assets/5047891/24297611/a6d73be0-10a3-11e7-8f5c-8f60be7ed330.png)
